playbooks/.gitignore

```diff
@@ -0,0 +1,6 @@
+*.log
```

Comment:
Can we consolidate everything in this .gitignore into the main instance at the project root? I think I brought most things up, but may have missed a few.